docs/docs_requirements.txt

```diff
@@ -17,7 +17,6 @@ djangorestframework==3.0.3
 docker-py==0.6.0
 gunicorn==19.1.1
 paramiko==1.15.2
-psycopg2==2.5.4
```

Comment:
I think everything else in the first block is imported directly by our code, so the other packages need to remain. (`psycopg2` was imported indirectly, as a string value at runtime.) Same for the client packages and other items in `doc_requirements.txt`.

We could try to see if there are others we can remove, but it's quite tedious to try:
```console
$ # first remove a package from docs_requirements.txt, then
$ rm -rf docs/venv
$ make -C docs test
```

I'm fairly confident we can't remove any others, but we should test that assumption at some point.